docs/guides/managing_packages.md

```diff
@@ -0,0 +1,122 @@
+
+```bash
+autonomy init --reset --author john_doe --remote --ipfs --ipfs-node "/dns/registry.autonolas.tech/tcp/443/https"
+```
+
+Now we are in the position of creating our first agent:
+
+```bash
+autonomy create my_agent
+```
+
+This agent will have some dependencies: some packages might be already available in the remote registry and some will need to be developed. Let's say, for example, that it will need the ability to communicate using the ACN protocol. Since that package is already available on the Autonolas IPFS registry, we'll use the `add command`:
+
+```bash
+cd my_agent/
+# Remote flag is not needed here as we initialized the default registry to remote
+autonomy add protocol valory/acn:0.1.0:bafybeidetrrkvdgveu4ph5g6v53lbh7ardfspbkpstmjxctx647bzyosyy
+```
+
+Now we might want to develop our own package, for example a cool skill. We could do it writing all the skill structure from scratch, but fortunately the CLI also provides a scaffolding option that will create a lot of boilerplate for us using its specification. For this example, we will use the specification from the [hello world skill](https://raw.githubusercontent.com/valory-xyz/open-autonomy/main/packages/valory/skills/hello_world_abci/fsm_specification.yaml). Download it to the agent directory and run:
+
+```bash
+autonomy scaffold fsm my_skill --spec fsm_specification.yaml
+```
+
+This will add your skill's boilerplate code to `my_agent/skills/my_skill` and some dependencies to the vendor directory. You can learn more on scaffolding by reading the [scaffolding guide](../guides/scaffolding.md).
+
+Now that we have all we need, publish the new agent and all its dependencies to the local registry:
+
+```bash
+autonomy publish --local --push-missing
+```
+
+The agent and its dependencies live now in the local registry, so we can delete the temporary agent:
+
+```bash
+cd ..
+autonomy delete my_agent
+```
+
+We will focus now on developing inside the local registry. First of all, it is useful to add other vendor paths to the `.gitignore` (for example `packages/valory` and `packages/open-aea`) so only your packages are tracked. Secondly, even if the skill is now in the local registry, there is another task we need to perform: update the `packages/packages.json` file. This file contains a list of all packages in the registry, as well as their hashes. It's useful to detect when packages change, are added or removed. If you open that file you will see it is empty. To reflect our latest changes, run:
+
+```bash
+autonomy packages lock
+```
+
+This will add the new skill hashes to it:
+
```

Comment:
will the below stay in sync? (i,e, is it covered by your hash update script? If not, then remove.